tests/test_plugins.py

```diff
@@ -18,11 +15,15 @@
 from variantlib.models.variant import VariantProperty
 from variantlib.protocols import PluginType
 from variantlib.protocols import VariantFeatureConfigType
+from variantlib.validators import ValidationError
 
 if sys.version_info >= (3, 10):
-    from importlib.metadata import EntryPoint
+    pass
 else:
-    from importlib_metadata import EntryPoint
+    pass
+
+
+RANDOM_STUFF = 123
```

Comment:
I don't think this variable is used anywhere

Reply:
https://github.com/wheelnext/variantlib/blob/392a9b260a52de54648abcc96bb4a5a39cea5bca/tests/test_plugins.py#L201

Reply:
My bad